backend/src/main.py

```diff
@@ -58,6 +60,10 @@ def main():
     parser.add_argument('white_row', help=WHITE_ROW_HELP)
 
     parser.add_argument('images', nargs='+', help=IMAGES_HELP)
+
+    # Optional arg to override default output directory (current working)
+    parser.add_argument('--output', required=False, help=OUTPUT_HELP)
```

Comment:
It would be better to name this something like --outpath to avoid confusion about what is being changed. --output could imply output type or single output.